Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -104,18 +104,24 @@ public struct NotificationMarkReadEvent: ChannelSpecificEvent, HasUnreadCount {
 }
 
 /// Triggered when a channel the current user is member of is marked as unread.
-public struct NotificationMarkUnreadEvent: ChannelSpecificEvent, HasUnreadCount {
+public struct NotificationMarkUnreadEvent: ChannelSpecificEvent {
     /// The current user.
     public let user: ChatUser
 
     /// The read channel identifier.
     public let cid: ChannelId
 
-    /// The unread counts of the current user.
-    public let unreadCount: UnreadCount?
-
     /// The event timestamp.
     public let createdAt: Date
+
+    /// The id of the first unread message id
+    public let firstUnreadMessageId: MessageId
+
+    /// The timestamp of the last read message
+    public let lastReadAt: Date
+
+    /// The number of unread messages for the channel
+    public let unreadMessages: Int
```

Comment:
What about making it more clear that it is a number?
```suggestion
    public let unreadMessagesCount: Int
```